Brewfile

```diff
@@ -4,7 +4,8 @@ tap "homebrew/core"
 tap "homebrew/services"
 tap "puma/puma"
 
-brew "postgresql@13", restart_service: true
+brew "postgresql@14", restart_service: true
```

Comment:
Initially went up to Postgres 15, but pgvector was overriding it and installing 14 because of its dependency https://formulae.brew.sh/formula/pgvector

Reply:
Note: it should be 15, need to install pgvector via the following steps https://www.notion.so/valiant/Upgrading-Postgres-1833c3f8bba342a789198fc806d35227?pvs=4#9f933270025a49af9464a62c46cadb7d